### PR TITLE
Add view

### PR DESCRIPTION
**Pull request by sangwa7**
# Hello reviewer ✋🏾

**Below are the tasks implemented:**

- [ ]  added view for users, posts, like, and comment
- [ ]  fixed linter errors

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
```
Comment (andersonlebon) [inline config/routes.rb]: - [ ] Kindly defines the root path route ("/") here(`root "users#index"`)

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <% @recent_posts.each do |post| %>
+        <%= render partial: "posts/post_card" , locals: { post: post } %>
+          <% end %>
+    </div>
+
+    <%= link_to "See all posts" , user_posts_path(@user) if @user.posts_counter> 0 %>
+      <%= link_to "Add a new post" , new_user_post_url(@user) %>
+
+        <%=form_with model: User.new, url: destroy_user_session_path, method: :delete, data: { turbo: false }  do %> <button class="custom-button">Sign out</button>
+          <% end %>
```
Comment (andersonlebon) [inline app/views/users/show.html.erb]: image

- [ ] Kindly fix this error accruing on this page.

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+      <strong>
+        <%= link_to user_path(comment.user) do %>
+          <%= comment.user.name %>
+        <% end %>
+        :
+      </strong>
+      <%= comment.text %>
+    </div>
+  <% end %>
+
```
Comment (andersonlebon) [inline app/views/posts/show.html.erb]: - [ ] KIndly fix the error on this page

image

File: app/views/posts/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,9 @@
+<% if @posts.present? %>
+<%= render(partial: 'users/user_card', locals: {user: @posts.first.user}) %>
+<% else %>
+This user doesn't have any post yet!
+<% end %>
+
+<% @posts.each do |post| %>
+  <%= render partial: 'post_card', locals: { post: post, comments: post.recent_five_comments} %>
+<% end %>
```
Comment (andersonlebon) [inline app/views/posts/index.html.erb]: image

- [ ] Kindly fix the error on this route

**Review by andersonlebon — CHANGES_REQUESTED** [review]
## Hi @sangwa7 

Good job so far! I know you have worked hard on making this milestone But  There are some issues that you still need to work on to go to the next project but you are almost there!

almost-there
## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
## Required Changes ♻️

- [ ] Kindly check comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by uwadonat — APPROVED** [review]
## Hi @sangwa7,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**